### ISSUE TITLE
GLM 模型缺失操作

### ISSUE THREAD

**Issue by lcylcy** [Oneflow-Inc/oneflow#6424]
```
#bool索引的赋值操作
def p1():
    a = np.random.rand(4,4)

    oi = flow.Tensor(a)
    ms = oi>0.5

    oi[ms] = 0
    print(oi)

```
#nn.functional.embedding函数缺失
def p4():
    from oneflow.nn.parameter import Parameter
    class a(flow.nn.Module):
        def __init__(self) -> None:
            super(a,self).__init__()
            self.weight = Parameter(flow.Tensor(1000,200))
        def forward(self,input):
            out = flow.nn.functional.embedding(
                input,
                self.weight,
                None,
                None,
                2,
                False,
                False
                )
            return out
    model = a()
    input = flow.randint(0,20,(30,50))
    out = model(input)
    print(out.shape)

**Comment by lcylcy** [comment]
```
#nn.functional.embedding函数缺失
def p4():
    from oneflow.nn.parameter import Parameter
    class a(flow.nn.Module):
        def __init__(self) -> None:
            super(a,self).__init__()
            self.weight = Parameter(flow.Tensor(1000,200))
        def forward(self,input):
            out = flow.nn.functional.embedding(
                input,
                self.weight,
                None,
                None,
                2,
                False,
                False
                )
            return out
    model = a()
    input = flow.randint(0,20,(30,50))
    out = model(input)
    print(out.shape)

**Comment by lcylcy** [comment]
#flow.numel函数缺失
def p5():
    a = np.random.rand(3,4)

    ti = torch.Tensor(a)
    print(torch.numel(ti))

    oi = flow.Tensor(a)
    print(flow.numel(oi))

**Comment by lcylcy** [comment]
这个也支持一下，我写了一下
```
#nn.functional.linear()
def p7():
    input = flow.rand(1,2,3)
    weight =flow.nn.parameter.Parameter(flow.rand(4,3))
    bias = flow.nn.parameter.Parameter(flow.rand(4))
    
    if input.dim() == 2 and bias is not None:
        ret = flow.addmm(bias,input,flow._C.transpose(weight,(1,0)))
        print(ret.shape)
    else:
        res = flow._C.matmul(input, weight, transpose_a=False, transpose_b=True)
        if bias is not None:
            res += bias
        print(res.shape)
p7()

**Comment by BBuf** [comment]
> 这个也支持一下，我写了一下
> 
> ```
> #nn.functional.linear()
> def p7():
>     input = flow.rand(1,2,3)
>     weight =flow.nn.parameter.Parameter(flow.rand(4,3))
>     bias = flow.nn.parameter.Parameter(flow.rand(4))
>     
>     if input.dim() == 2 and bias is not None:
>         ret = flow.addmm(bias,input,flow._C.transpose(weight,(1,0)))
>         print(ret.shape)
>     else:
>         res = flow._C.matmul(input, weight, transpose_a=False, transpose_b=True)
>         if bias is not None:
>             res += bias
>         print(res.shape)
> p7()
> ```

这个具体是哪里不支持

**Comment by lcylcy** [comment]
这个我自己写的，之后可以合并到master上面。。.只是目前oneflow没有F.linear()

**Comment by BBuf** [comment]
> 这个我自己写的，之后可以合并到master上面。。.只是目前oneflow没有F.linear()

那你就在表格里面加一下这个需求
